### ISSUE TITLE
Windows meta key

### ISSUE THREAD

**Issue by ihandley** [holgerbrandl/r4intellij#178]
I'm looking for the equivalent of the _Run selected lines_ command (Ctrl+Enter) from R Studio. I believe it's the _Evaluate in R Console_ option triggered by the keyboard shortcut Meta+Alt+Enter. But for the life of me, I cannot figure out how to use the _Meta_ key. I've looked at File > Settings > Keymap and don't find anything there indicating which key is mapped to Meta.

I'm sure this is something super simple, but point me in the right direction?

**Comment by holgerbrandl** [comment]
Isnt't meta == Ctrl on windows?

**Comment by ihandley** [comment]
That's what I originally thought, but Ctrl+Alt+Enter does not trigger the _Evaluate in R Console_ command.

**Comment by holgerbrandl** [comment]
Indeed, same here.  See https://stackoverflow.com/questions/32921994/how-to-make-windows-key-the-intellij-idea-command-meta-key-under-windows

So I guess the easiest would be to add a custom keybinding under Settings->Keymap->`Evaluate in R Console`
![image](https://user-images.githubusercontent.com/200952/45828027-eb92a380-bcf7-11e8-9a15-4b0238e6f6ec.png)

**Comment by ihandley** [comment]
That suggestion would most definitely work, and I think it's the best solution considering that command seems to be the only one utilizing the _Meta_ key. Is there any reason to not unbind Ctrl+Enter from _Split Line_? It seems to be doing the same thing as just regular Enter.

**Comment by AmineI** [comment]
> That suggestion would most definitely work, and I think it's the best solution considering that command seems to be the only one utilizing the Meta key. Is there any reason to not unbind Ctrl+Enter from Split Line? It seems to be doing the same thing as just regular Enter.

Well, regular Enter sends the cursor to the new line while Ctrl+Enter does not. But I agree that it's not much of a difference.
